AENS.md

```diff
@@ -297,18 +303,25 @@ A `claim` transaction MUST NOT be in included in the same block as its
  ------------ ----
 | client_ttl | 8  |
  ------------ ----
+| rent       | 32 |
+ ------------ ----
 | pointers   |    |
  ------------ ----
 ```
 
-The `update` transaction MUST be signed by the owner
-of the name entry to be updated.
+The `update` transaction MUST be signed by the owner of the name entry to be
+updated.
 
-The `expire_by` MUST NOT be more than 36000 blocks into
-the future.
+The `expire_by` MUST NOT be more than `floor(60 * 24 * (365/2)/key_block_interval)`,
+i.e. half a year, blocks into the future.
 
 `update` transaction may be used to extend the lease of the name.
-We do not require an additional fee for extending the lease.
+
+`rent` is the reoccurring fee that has to be paid for names. Our target is to
```

Comment:
Do I understand it correctly that the total amount (i.e. considering all blocks the name is rented for) needs to be charged all at time of inclusion of this update transaction?

Reply:
That is the only time we can know if the transaction creator has funds, so I guess there is no other alternative.

Reply:
Rent is charged up front, yes